.github/workflows/test-application.yaml

```diff
@@ -35,8 +35,8 @@ jobs:
                       mysql-version: '8.0'
                       create-project: false
                       create-database: false
-                      checkout-directory: 'project'
-                      working-directory: 'project'
+                      checkout-directory: './'
```

Comment:
this does fix the normal test on 7.4 and 8.0 with `create-project: false`

Reply:
i think this happens because the `working-dir` option of the `composer-install` action was moved: https://github.com/ramsey/composer-install/pull/107

Reply:
so i think we should adjust the options of the action 🙂